demo/app/ios/Runner/flutterPlugin.swift

```diff
@@ -502,6 +592,33 @@ public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
         
     }
     
+    
+    
+    public func requestCredentialWithWalletInitiatedFlow(redirectURIWithParams: String, result: @escaping FlutterResult){
+        guard let walletInitiatedOpenID4CI = self.walletInitiatedOpenID4CI else{
+            return  result(FlutterError.init(code: "NATIVE_ERR",
+                                             message: "error while process requestCredential auth",
```

Comment:
Similar comment regarding errors - add wrapping instead of repeating everywhere inside the method